src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/mappers/PartyDetailsMapper.java

```diff
@@ -42,4 +49,40 @@ public List<PartyDetailsModel> map(
 
         return partyDetails;
     }
+
+    public static PartyDetailsModel appendBookingStatus(Optional<InterpreterBookingStatus> spokenBookingStatus,
```

Comment:
Seems to be missing unit test for this method?

Reply:
I didn't add a unit test for this because it's used as a static method in AppellantDetailsMapper and WitnessDetailsMapper and their tests already cover it fully.
The lines of this method are already green on Sonar: [Sonar PR-230 PartyDetailsMapper.java](https://sonarcloud.io/component_measures?metric=new_coverage&selected=IAHEARINGSAPI%3Asrc%2Fmain%2Fjava%2Fuk%2Fgov%2Fhmcts%2Freform%2Fiahearingsapi%2Fdomain%2Fmappers%2FPartyDetailsMapper.java&view=list&pullRequest=230&id=IAHEARINGSAPI)

Reply:
I see what you're saying, but ideally unit tests should be in their own places for separation of concerns so that it's clear  where each functionality is tested, it's not just about test coverage but also for maintainability. (for integration testing though it's okay to mix functionality from different places)